plugins/destination/sqlite/client/migrate.go

```diff
@@ -88,7 +88,7 @@ func (c *Client) autoMigrateTable(_ context.Context, table *schema.Table) error
 		switch {
 		case sqliteColumn == nil:
 			c.logger.Debug().Str("table", table.Name).Str("column", col.Name).Msg("Column doesn't exist, creating")
-			sql := "alter table " + table.Name + " add column " + columnName + " " + columnType
+			sql := "alter table \"" + table.Name + "\" add column \"" + columnName + "\" \"" + columnType + `"`
```

Comment:
Why not use \` throughout and get rid of `"`
```suggestion
			sql := `alter table "` + table.Name + `" add column "` + columnName + `" "` + columnType + `"`
```